src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -133,9 +133,16 @@ internal static ProcessInfo CreateProcessInfo(Interop.procfs.ParsedStat procFsSt
                 ProcessId = pid,
                 ProcessName = procFsStat.comm,
                 BasePriority = (int)procFsStat.nice,
-                VirtualBytes = (long)procFsStat.vsize,
-                WorkingSet = procFsStat.rss * Environment.SystemPageSize,
                 SessionId = procFsStat.session,
+                PoolPagedBytes = (long)procFsStatus.vmswap,
+                VirtualBytes = (long)procFsStatus.vmsize,
+                VirtualBytesPeak = (long)procFsStatus.vmpeak,
+                WorkingSetPeak = (long)procFsStatus.vmhwm,
+                WorkingSet = (long)procFsStatus.vmrss,
+                PageFileBytes = (long)procFsStatus.vmswap,
+                PrivateBytes = (long)procFsStatus.vmdata,
```

Comment:
Are there any tests we're skipping because these weren't set and can now enable?  Or tests we can add?  This stuff is always hard to validate, though.

Reply:
I'll look into that.